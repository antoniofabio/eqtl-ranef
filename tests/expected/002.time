	Command being timed: "./test-002.sh"
	User time (seconds): 181.87
	System time (seconds): 1.19
	Percent of CPU this job got: 21%
	Elapsed (wall clock) time (h:mm:ss or m:ss): 14:18.15
	Average shared text size (kbytes): 0
	Average unshared data size (kbytes): 0
	Average stack size (kbytes): 0
	Average total size (kbytes): 0
	Maximum resident set size (kbytes): 809312
	Average resident set size (kbytes): 0
	Major (requiring I/O) page faults: 0
	Minor (reclaiming a frame) page faults: 103285
	Voluntary context switches: 5908
	Involuntary context switches: 229237
	Swaps: 0
	File system inputs: 0
	File system outputs: 7672
	Socket messages sent: 0
	Socket messages received: 0
	Signals delivered: 0
	Page size (bytes): 4096
	Exit status: 0
