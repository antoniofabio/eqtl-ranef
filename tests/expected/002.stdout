outcome103	regressor2	0.0325365623227719
outcome11	regressor1	0.033541147320197
outcome130	regressor2	0.050405002241782
outcome142	regressor2	0.0894473589886824
outcome144	regressor2	0.0862931878322179
outcome147	regressor2	0.0607942019846332
outcome14	regressor2	0.0698238767059064
outcome152	regressor1	0.00394689242316357
outcome159	regressor2	0.0827822583222105
outcome164	regressor2	0.0517480106645231
outcome165	regressor1	0.0596990309057334
outcome166	regressor2	0.0532927604353382
outcome169	regressor1	0.0626251118097336
outcome199	regressor1	0.0349360944043778
outcome208	regressor2	0.0356766696939756
outcome225	regressor2	0.021991405334855
outcome228	regressor1	0.071017367645301
outcome234	regressor1	0.0359061115822172
outcome234	regressor2	0.0587896835980864
outcome236	regressor1	0.0330120809620878
outcome238	regressor2	0.068902658499351
outcome240	regressor2	0.0777786186905564
outcome242	regressor1	0.0564002995986698
outcome242	regressor2	0.060349122900411
outcome249	regressor1	0.00138082463148325
outcome253	regressor2	0.0350100544544206
outcome254	regressor1	0.0211542302870382
outcome256	regressor2	0.0802286607247205
outcome267	regressor1	0.0925271200855481
outcome269	regressor2	0.0681203775322091
outcome271	regressor2	0.00377797601672558
outcome273	regressor1	0.0641410807872831
outcome276	regressor2	0.0574592786140935
outcome282	regressor2	0.043210158959617
outcome28	regressor2	0.033548206647807
outcome301	regressor2	0.0900445239332451
outcome302	regressor1	0.0900741080738507
outcome305	regressor1	0.0615853007902517
outcome307	regressor2	0.0979408374825312
outcome309	regressor1	0.0789333143640794
outcome30	regressor1	0.0462759764616038
outcome316	regressor1	0.0489805441570392
outcome316	regressor2	0.0955394810741442
outcome317	regressor2	0.0700524998357606
outcome319	regressor2	0.0114470343508347
outcome31	regressor1	0.084072808989726
outcome31	regressor2	0.0680778536232788
outcome321	regressor1	0.087392671767382
outcome324	regressor2	0.0127974593254924
outcome332	regressor1	0.0811678975593089
outcome332	regressor2	0.0345143233829059
outcome334	regressor1	0.0680516093753948
outcome338	regressor1	0.0661107222897772
outcome348	regressor1	0.0342138125734546
outcome354	regressor2	0.0106643154291089
outcome35	regressor2	0.0522394411970028
outcome362	regressor2	0.0378778532753085
outcome369	regressor2	0.0918429574376158
outcome370	regressor2	0.00938654527654634
outcome382	regressor2	0.088362055245545
outcome385	regressor1	0.0255969474917829
outcome387	regressor1	0.0888359061613241
outcome393	regressor1	0.0834845703062013
outcome393	regressor2	0.00392260882006368
outcome396	regressor1	0.0684493017324679
outcome398	regressor2	0.00841852712789738
outcome401	regressor1	0.0421432337147575
outcome403	regressor1	0.0916943876320012
outcome404	regressor2	0.0224563650724841
outcome406	regressor2	0.0551687914217282
outcome413	regressor2	0.00441883550201094
outcome416	regressor2	0.00222730626418183
outcome421	regressor2	0.026064441657946
outcome429	regressor1	0.0877774089432111
outcome434	regressor2	0.008145322439358
outcome436	regressor2	0.0830663797742703
outcome439	regressor1	0.00185922952488431
outcome443	regressor1	0.0944478989257502
outcome446	regressor2	0.0679009288950019
outcome451	regressor2	0.0725287170302779
outcome457	regressor1	0.0439239305382586
outcome458	regressor2	0.0951873626459277
outcome462	regressor1	0.0227312527607727
outcome466	regressor2	0.0394549669258231
outcome469	regressor1	0.0514410978685054
outcome488	regressor1	0.076413553290723
outcome500	regressor1	0.0635697864326271
outcome504	regressor1	0.0989327728049506
outcome506	regressor2	0.0156373806696265
outcome50	regressor2	0.0390156898426308
outcome514	regressor2	0.037577640532011
outcome520	regressor1	0.0995824417238426
outcome521	regressor2	0.0586471887818158
outcome525	regressor1	0.0929730483849698
outcome525	regressor2	0.0928510641153085
outcome527	regressor1	0.0308809260393631
outcome537	regressor2	0.0232805192845352
outcome541	regressor2	0.00841582098278195
outcome547	regressor2	0.0323485706730985
outcome549	regressor1	0.0203096112693548
outcome549	regressor2	0.0366887437353146
outcome554	regressor1	0.0756822581942527
outcome559	regressor1	0.042898301319244
outcome563	regressor1	0.00161512881679328
outcome568	regressor1	0.0956073710463073
outcome569	regressor2	0.0334383957170268
outcome56	regressor1	0.0376561476395674
outcome56	regressor2	0.0199907526878669
outcome570	regressor1	0.0379573695027463
outcome576	regressor2	0.0909747803773497
outcome578	regressor1	0.0172211275325164
outcome579	regressor1	0.0975830693309091
outcome582	regressor2	0.0642283644785473
outcome590	regressor2	0.0500400888384925
outcome592	regressor2	0.0866570052257551
outcome595	regressor2	0.0207691581150641
outcome597	regressor2	0.0993382472650093
outcome606	regressor2	0.07507760483899
outcome609	regressor2	0.0933934956413671
outcome618	regressor2	0.0107667882326525
outcome624	regressor1	0.0268406605210252
outcome625	regressor1	0.0578141059915856
outcome626	regressor1	0.0239479998852488
outcome626	regressor2	0.0693274481348567
outcome628	regressor2	0.024548362329945
outcome631	regressor1	0.0428961654823623
outcome641	regressor1	0.0404950335076071
outcome642	regressor2	0.0528153075308043
outcome644	regressor2	0.00117402772046939
outcome652	regressor2	0.0759904450303129
outcome659	regressor1	0.0910752804847771
outcome65	regressor1	0.0351588630105254
outcome665	regressor1	0.0999862992400272
outcome666	regressor1	0.00812949139761844
outcome672	regressor2	0.0266696023793339
outcome674	regressor2	0.0142050679893047
outcome676	regressor2	0.0231038578082265
outcome677	regressor1	0.056554247564005
outcome685	regressor1	0.056123015138911
outcome694	regressor1	0.0551653139258192
outcome697	regressor1	0.0255513695103154
outcome700	regressor1	0.0552058968838734
outcome703	regressor2	0.0979739794096469
outcome706	regressor1	0.0973154942234874
outcome706	regressor2	0.017532005072146
outcome713	regressor1	0.0163137815382378
outcome715	regressor2	0.0225214472260048
outcome717	regressor2	0.0678800113192437
outcome719	regressor2	0.0403909461866265
outcome721	regressor1	0.0254889365170573
outcome728	regressor2	0.0978760595201232
outcome731	regressor1	0.0594756159426405
outcome731	regressor2	0.0233290405362729
outcome733	regressor2	0.0656129436414464
outcome738	regressor1	0.0303044258367373
outcome73	regressor2	0.081096562843067
outcome742	regressor2	0.0807941415449
outcome744	regressor1	0.0995329228261456
outcome744	regressor2	0.0737326460446236
outcome749	regressor2	0.0318272505736934
outcome755	regressor2	0.0940417562539475
outcome757	regressor1	0.0250956958928922
outcome764	regressor1	0.0733426996628901
outcome769	regressor1	0.0306327853933688
outcome772	regressor1	0.0596372943948393
outcome780	regressor2	0.0425252745345017
outcome782	regressor1	0.0120294157680506
outcome793	regressor1	0.0924353871366202
outcome794	regressor2	0.0329187631890203
outcome800	regressor1	0.0388527239796559
outcome802	regressor1	0.0758747254208273
outcome805	regressor1	0.0938980427042515
outcome805	regressor2	0.0614904369484835
outcome828	regressor2	0.0994799038823672
outcome834	regressor2	0.00997294095011313
outcome835	regressor1	0.0940009604740974
outcome841	regressor2	0.0567348553338449
outcome844	regressor2	0.0534355789231332
outcome855	regressor1	0.0239018094684346
outcome85	regressor2	0.0454857907287943
outcome863	regressor2	0.0344146972461212
outcome866	regressor2	0.043089940661067
outcome872	regressor1	0.00972108798643541
outcome874	regressor2	0.0422411792150209
outcome878	regressor2	0.0523260405368736
outcome880	regressor2	0.0305288452968722
outcome884	regressor2	0.0417512879868961
outcome885	regressor1	0.0188110571822456
outcome886	regressor1	0.0769270557336066
outcome88	regressor1	0.0084558244739562
outcome891	regressor1	0.0133749000998277
outcome898	regressor1	0.0188643982674974
outcome899	regressor2	0.0270892778889649
outcome90	regressor1	0.000618545190970514
outcome911	regressor1	0.0100999476929507
outcome912	regressor1	0.0632018528492972
outcome913	regressor2	0.0289628804124557
outcome917	regressor1	0.0386405935084096
outcome920	regressor1	0.0158916632946754
outcome924	regressor1	0.0615703102558693
outcome930	regressor2	0.0667255298343387
outcome938	regressor1	0.0130168656169157
outcome939	regressor2	0.0798668828875971
outcome944	regressor1	0.0924246403940854
outcome948	regressor1	0.0807140557208745
outcome94	regressor1	0.0152686792701357
outcome951	regressor2	0.0630948402488141
outcome955	regressor1	0.0728167074193879
outcome95	regressor2	0.0693309676679492
outcome971	regressor2	0.0376880151418001
outcome973	regressor1	0.0958963692309035
outcome980	regressor2	0.0718686773590085
outcome984	regressor2	0.0267747701717827
outcome987	regressor2	1.35067218646505e-05
outcome990	regressor1	0.0309066638993854
outcome990	regressor2	0.0331857447354853
outcome999	regressor1	0.0505067562121754
outcome9	regressor1	0.0817019557699952
